kubernetes/ingress-nginx.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: extensions/v1beta1
```

Comment:
I think this file isn't needed an can be removed (you already have the `Ingress` in `opendebates.yaml.j2`).